src/components/Message.tsx

```diff
@@ -303,6 +304,15 @@ export default class Message extends React.Component<Props, State> {
             icon="delete"
           />
         )}
+
+        { !message.user.isSelf && <>
+          { this.props.isCompressedUser
+            ? <Menu.Item icon="maximize" text="Disable message compression" onClick={this.onMessageCompressionStopRequest} />
+            : <Menu.Item icon="minimize" text="Enable message compression" onClick={this.onMessageCompressionStartRequest} />
+          }
+          </>
+        }
+
```

Comment:
Weird formatting, not sure why Prettier didn't change this and ESLint even allowed it. I talk about this in a comment, I'll investigate and fix this so we can update the PR and re-run the process on it.